### DISCUSSION TITLE
Use Zen Glance by long-pressing the link

### DISCUSSION THREAD

**Discussion by Car0lina5** [zen-browser/desktop#2544] [Ideas]
The max focus extension has this option that I would like to use and I think it could be a good option. Well, that's my suggestion.

**Comment by ClixTW** [comment]
Glance is great, but it has to be activated via the keyboard, which makes me realize I hardly ever use it. If it could be activated by a long press, I would definitely uninstall Max Focus right away.

**Comment by BlueFox1616** [comment]
it is already planned it says its coming soon in the settings
